recipe/meta.yaml

```diff
@@ -56,10 +56,10 @@ requirements:
     - python >=3.7
     - ccache
     - libprotobuf =3.20.3 # for protoc compiler, libraries are used from libprotobuf-static
-    - flatbuffers =23.3.3
+    - flatbuffers >=23.3.3
     - sysroot_linux-64 =2.17  # [linux64]
   host:
-    - tbb-devel =2021.5 # starting from this version TBB uses hwloc in conda
+    - tbb-devel >=2021.5 # starting from this version TBB uses hwloc in conda
```

Comment:
If you feel the need to set a lower bound. You must leave an unbound tbb-devel for the pinning to take in effect. It’s a quirk of conda’s tooling

Reply:
removed